content/en/service_catalog/adding_metadata.md

```diff
@@ -428,6 +428,14 @@ As an alternative to the GitHub integration and Terraform, you can use an open-s
 
 This GitHub Action allows you to register your services with the Service Catalog using a GitHub Action, with full control over when this information is sent to Datadog, and implement other compliance checks unique to your organization.
 
+## IDE Plugins
+
+Datadog provides a [JSON Schema][17] for service definitions so that when you are editing a service definition in a [supporting IDE][18], features such as autocomplete and validation are provided. The [JSON schema for Datadog service definitions][19] is registered with the open source [Schema Store][18].
+
+The plugin supports the new v3.0 schema and includes information that will tell your IDEs which filenames it applies to—specifically, the files entity.datadog.yaml, entity.datadog.yml, and entity.datadog.json to quickly and accurately complete your entity definition file without referencing documentation. In the example below, the IDE automatically creates keys as you fill out the entity definition.
```

Comment:
We say "plugin" here (and "plugin" is in the title as well), but it's unclear to me what exactly this refers to. Is the Datadog JSON Schema a plugin? I would suggest using plugin everywhere or using some other word like "integration". Whatever you think is best, and just stay consistent.

Reply:
Also, I'm not sure how much value-add it is to include "supports the new v3.0 schema"? If you do think this is important, then I'd ask you to clarify what this refers to -- is v3.0 the version for JSON Schema or for Datadog's JSON Schemas? (Let me know if my question doesn't make sense.)